### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by Calebyang93**
Updated Nginx and docker container integration for users to understand how to effectively utilize the project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,12 +18,6 @@
     "@open-wc/rollup-plugin-html": "^0.3.4",
     "@rollup/plugin-node-resolve": "^7.1.1",
     "babel-plugin-bundled-import-meta": "^0.3.2",
-    "es-dev-server": "^1.45.2",
```
Comment (thepassle) [inline package.json]: Dont remove the dev server please, we need this

Comment (Calebyang93) [reply package.json]: Ok Pascal, I have added the es-dev-server extension back in the package.json file and have pushed it.

File: README.md [hunk 2]
```diff
@@ -35,3 +35,20 @@ npm run start:build
  Nodejs documentation
  https://nodejs.org/en/docs/
 
+# Nginx
+NginX is used on this project. 
+For utilization of NginX in this project, one familiarize themselves with the mechanics of NginX.
+http://nginx.org/en/docs/
+It is important for people to note how NginX unit tests are conducted.
```
Comment (thepassle) [inline README.md]: is this really relevant? do we even use nginx unit tests?

**Comment by thepassle** [comment]
closing this due to inactivity/relevancy

**Pull request closed without merging**